design/admin/templates/ajaxuploader/location.tpl

```diff
@@ -3,6 +3,20 @@
     <fieldset>
         <legend>{"Step 2/3: Choose a location for the new '%class' object"|i18n( 'design/admin/ajaxuploader', '', hash( '%class', $class.name|wash() ) )}</legend>
         <p>{'<em>%file</em> has successfully been uploaded.'|i18n( 'design/admin/ajaxupload', '', hash( '%file', $file.original_filename|wash() ) )}</p>
+        {if $class.identifier|eq( 'image' )}
+            {def $alt_text_required = ''}
```

Comment:
Boolean no longer used due to restructuring the code to support stronger messaging around the alternative text field.